modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.h

```diff
@@ -164,6 +164,9 @@ class SOFA_GENERAL_ANIMATION_LOOP_API MechanicalMatrixMapper : public MixedInter
 
     Data<helper::vector<std::string>> d_forceFieldList;
     SingleLink < MechanicalMatrixMapper<DataTypes1, DataTypes2>, sofa::simulation::Node , BaseLink::FLAG_STOREPATH > l_nodeToParse;
+    Data <bool> l_stopAtNodeToParse;
```

Comment:
The Sofa convention is to prefix Data field with a d_

Reply:
Thank you, that's not a link anyway!